neo4j/README.md

```diff
@@ -3,11 +3,14 @@
 ## Overview
 
 This check monitors [neo4j][1] through the Datadog Agent.
+In [Neo4j 5][9], all metric names now include dbms or database namespaces.
+
+This check has been tested against Neo4j 4 and Neo4j 5 databases. If there are any issues, contact DataDog support or support@neo4j.com.
+
```

Comment:
Overview should include:

1. What neo4j does in 1-2 sentences 
2. The value of this integration
3. What type of data is being collected - you can specify out each version and what is collected

Ex: 
With this integration, you can collect both Neo4j and Neo4j version 5 metrics to monitor abc.

Reply:
Updated. Thanks.

Reply:
Created 2 default dashboards. Probably not the most beautiful dashboards that you have ever seen.